fix various delegate docs (#635)
harness/developer-hub: docs/platform/2_Delegates/manage-delegates/select-delegates-with-selectors.md

### Review: Delegate tags

> A delegate tag with the same name as your delegate is added to your delegate automatically when you set it up in Harness. You can also give additional comma separated tags in the helm command line or in kubernetes yaml file like below

Reviewer: ```suggestion
A delegate tag with the same name as your delegate is automatically added to your delegate during the configuration process. You can add one or more comma-separated tags on the `helm` command line or in the Kubernetes YAML file, as shown in the following example.
```

```
...  
    env:
    ....
    - name: DELEGATE_TAGS
      value: "tag1,tag2"
...
```

> You can also add more tags in the **Tags** field during the setup process:

Reviewer: ```suggestion
You can also add tags to the **Tags** field during the setup process:
```
